src/backend/src/services/recruitment.services.ts

```diff
@@ -63,6 +63,45 @@ export default class RecruitmentServices {
   }
 
   /**
+   * Edits the FAQ
+   * @param question the updated question value
+   * @param answer the updated answer value
+   * @param faqId the requested FAQ to be edited
+   * @param submitter the user editing the FAQ
+   * @param organizationId the organization the user is currently in
+   * @returns the updated FAQ
+   */
+  static async editFAQ(
+    question: string,
+    answer: string,
+    submitter: User,
+    organizationId: string,
+    frequentlyAskedQuestionId: string
+  ) {
+    if (!organizationId) {
```

Comment:
Dont check if the organization id doesn't exist, first find the organization and check if that doesn't exist. look at how its done in other endpoints